packages/atomic/src/components/common/pager/pager-button.tsx

```diff
@@ -0,0 +1,77 @@
+import {FunctionalComponent, h} from '@stencil/core';
```

Comment:
Since the file contains all pager button definitions, I'd rename it to "pager-buttons.tsx"